### PR TITLE
B/card UI changes - Row expandable cards

### PR DESCRIPTION
**Pull request by memacius32**
Refactor Farm cards into compact rows. 
- Shrunk the height and spaces in general.
- Remove spaces between cards, made them as a list.
- Expand the card on click. (won't show anything atm in desktop).
- Move Liquidity data and Actions - Deposit/Manage&Claim Buttons - to expandable part of the card when On Mobile.
- If viewport is Mobile, and user is staking, automatically expand the card (to show claim/manage actions).

Todo's for other PR's:
- Move Actions & other data (Add liquidity link, show user Staked) etc to expandable content.
- Remove Actions as they are now in favour of showing all the Manage page and Claim modal content on the expandable section.
- To be discussed: Remove farm sections and a tag  showing the farm category in the card. 

#### General Preview: ####

https://user-images.githubusercontent.com/96993065/179551953-262c90c7-ea13-4fe5-b10c-ac20f5137366.mp4

#### Preview of cards opening if Staking & Mobile (Manage/Claim are on the expandable section): ####

https://user-images.githubusercontent.com/96993065/179551833-85734378-fe85-4494-be1a-aa2ef38e421c.mp4

### Screens ###
Screen Shot 2022-07-18 at 11 54 28
Screen Shot 2022-07-18 at 12 41 42
Ï

### CODE REVIEW

File: src/components/earn/PoolCardTri/index.tsx [hunk 1]
```diff
@@ -70,16 +79,22 @@ const DefaultPoolCardtri = ({
   const history = useHistory()
   const { t } = useTranslation()
 
+  const [showMore, setShowMore] = useState(isMobileOnly && isStaking ? true : false)
```
Comment (specialist0x) [inline src/components/earn/PoolCardTri/index.tsx:82]:
```suggestion
  const [showMore, setShowMore] = useState(isMobileOnly && isStaking)
```

File: src/components/earn/PoolCardTri/index.tsx [hunk 2]
```diff
@@ -103,6 +118,23 @@ const DefaultPoolCardtri = ({
     )
   }
 
+  function renderActionsContainer() {
```
Comment (specialist0x) [inline src/components/earn/PoolCardTri/index.tsx:121]: IMO This should be its own component

File: src/components/earn/PoolCardTri/index.tsx [hunk 3]
```diff
@@ -111,42 +143,51 @@ const DefaultPoolCardtri = ({
...
+        <DetailsContainer>
+          {showMore ? (
+            <>
+              <ChevronUp size="15" />
+            </>
+          ) : (
+            <>
+              <ChevronDown size="15" />
+            </>
+          )}
```
Comment (specialist0x) [inline src/components/earn/PoolCardTri/index.tsx:175]:
```suggestion
          {showMore ? (
            <ChevronUp size="15" />
          ) : (
            <ChevronDown size="15" />
          )}
```

File: src/components/earn/PoolCardTri/index.tsx (hunk 1 above)
Comment (specialist0x) [reply src/components/earn/PoolCardTri/index.tsx:82]: If mobile UI is constrained vertically, doesn't it make sense to have everything collapsed?  This is assuming it's easy to tell which pools the user is staked in while the card is collapsed

**Review by specialist0x — APPROVED** [review]
Approved with comments

**Comment by memacius32** [comment]
Addressed suggestions and merging into base branch for farm cards changes

**Pull request merged**